addons/account/i18n/account.pot

```diff
@@ -9527,18 +9527,12 @@ msgstr ""
 
 #. module: account
 #. odoo-python
-#: code:addons/account/models/account_move.py:0
+#: code:addons/account/models/account_journal_dashboard.py:0
 msgid ""
 "No journal could be found in company %(company_name)s for any of those "
 "types: %(journal_types)s"
 msgstr ""
 
-#. module: account
-#. odoo-python
-#: code:addons/account/models/account_journal.py:0
-msgid "No journal found"
-msgstr ""
```

Comment:
Removing translation terms in stable is bad ; avoid this diff (the term will stay there unused ; it's fine)